src/lib/utils/theme/typography/Typography.ts

```diff
@@ -0,0 +1,105 @@
+import { TypographyOptions } from '@mui/material/styles/createTypography'
+import { Palette } from '@mui/material/styles/createPalette'
+
+export const typographyOptions: (
```

Comment:
Moved Typography styles here. They get added directly to the MUI theme and then we can just use MUI typography instead of our own component wrapper.